simple_history/models.py

```diff
@@ -24,19 +24,28 @@
 registered_models = {}
 
 
+def default_get_user(request, **kwargs):
+    try:
+        return request.user
+    except AttributeError:
+        return None
+
+
 class HistoricalRecords(object):
     thread = threading.local()
 
     def __init__(self, verbose_name=None, bases=(models.Model,),
                  user_related_name='+', table_name=None, inherit=False,
-                 excluded_fields=None,
-                 history_id_field=None,
+                 excluded_fields=None, history_id_field=None,
+                 user_model=None, get_user=default_get_user,
```

Comment:
Want to add `user_model` and `get_user` after `history_change_reason_field` in case anyone is just using ordered args?

Reply:
Sure I can re-order them